MarkBox/data/tables/markbox-sct.tbm

```diff
@@ -531,19 +547,17 @@ mn.LuaSEXPs["lua-mark-weapon"].Action = function(weapon, ship) MarkBox:AddWeapon
 mn.LuaSEXPs["lua-count-weapon"].Action = function(weapon) return MarkBox:GetCountWeapon(weapon) end
 mn.LuaSEXPs["lua-get-weapon-distance"].Action = function(weapon) return MarkBox:GetMarkedWeaponDistance(weapon) end
 
+mn.LuaSEXPs["lua-mark-set-flags"].Action = function(subsys_diagonals, subsys_health)
+	MarkBox.DrawSubsysDiagonals = subsys_diagonals
+	MarkBox.DrawSubsysHealth = subsys_health
+end
 ]
 
-$State: GS_STATE_GAME_PLAY
-$On Gameplay Start:
+$On Mission Start:
 [
 	MarkBox:Init()
 ]
 
-$On State End:
-[
-	MarkBox.Enabled = false
-]
-
```

Comment:
Whats the reasoning for not disabling markbox on mission end?
If thats intentional, it should likely be an additional optional parameter in the init function, that it'll not disable after the mission

Reply:
It's because this is `$On State End`, not `$On Mission End`.

I thought of changing it to `$On Mission End`, but a) the Enabled flag is always set to false via `$On Mission Start`, and b) there is nothing that runs outside of a mission that would be controlled by this flag.

Reply:
hmm, fair.